src/index.html

```diff
@@ -11,6 +11,21 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form data-qa="big" class="form_style">
+      <input
+        type="text"
+        placeholder="Try &quot;Los Angeles&quot;"
+        class="big_input genural_input_style"
```

Comment:
give more simple name for elements
```suggestion
        class="big_input input"
```